web/datapipeline/clusters_projector.go

```diff
@@ -49,14 +53,17 @@ func transformClusterData(cluster *cluster.Cluster) (*entities.Cluster, error) {
 		return nil, fmt.Errorf("no cluster ID found")
 	}
 
+	clusterDetail, _ := parseClusterDetails(cluster)
+	fmt.Println(clusterDetail)
 	return &entities.Cluster{
 		ID:          cluster.Id,
 		Name:        cluster.Name,
 		ClusterType: detectClusterType(cluster),
 		// TODO: Cost-optimized has multiple SIDs, we will need to implement this in the future
-		SIDs:            getHanaSIDs(cluster),
+		SIDs:            getClusterSIDs(cluster),
```

Comment:
I know that this was already here, but the SID is exclusively part of SAP cluster types.
For example, a DRBD cluster won't have any SID.
Here, to be purist, the SIDs should go in the details. Anyway, I understand we will only manage SAP cluster in any case.
It was just an informative comment

Reply:
basically they are here because they are used in filtering